### PR TITLE
Marvel.external.cluster

### PR DESCRIPTION
**Pull request by JasonIamAUnixAdmin**
Now deals with an external Marvel cluster for monitoring

### CODE REVIEW

File: elasticsearch/etc/elasticsearch/elasticsearch.yml [hunk 1]
```diff
@@ -61,7 +62,22 @@ path.logs: /mnt/elasticsearch/logs
 # Disable the JVM from being swapped out
 bootstrap.mlockall: true
 
-
+{%- if pillar.elasticsearch.marvel.install %}
+    {%- if pillar.elasticsearch.marvel.external_cluster %}
+        {%- if pillar.elasticsearch.version == "1.7" %}
```
Comment (clarkperkins) [inline elasticsearch/etc/elasticsearch/elasticsearch.yml]: should this be checking for 1.7 or just 1.x?

File: elasticsearch/kibana.sls [hunk 2]
```diff
@@ -106,6 +92,23 @@ install_marvel:
   - unless: 'test -d /usr/share/kibana/installedPlugins/marvel'
```
Comment (clarkperkins) [inline elasticsearch/kibana.sls:93]: We'll probably want to add a `require_in` clause for `file: /usr/share/kibana/optimize/.babelcache.json` just to ensure that it gets run after the marvel install

**Comment by JasonIamAUnixAdmin** [comment]
Made changes based on comments.

File: elasticsearch/etc/elasticsearch/elasticsearch.yml [hunk 3]
```diff
@@ -61,7 +61,22 @@ path.logs: /mnt/elasticsearch/logs
 # Disable the JVM from being swapped out
 bootstrap.mlockall: true
 
-
+{%- if pillar.elasticsearch.marvel.install %}
+    {%- if pillar.elasticsearch.marvel.external_cluster %}
+        {%- if pillar.elasticsearch.version.split('.'[1] | int < 2 %}
```
Comment (clarkperkins) [inline elasticsearch/etc/elasticsearch/elasticsearch.yml]: `pillar.elasticsearch.version.split('.')[0]`

**Comment by clarkperkins** [comment]
:+1:

**Pull request merged**